### PR TITLE
Winamp 5.91.0.10029

### PR DESCRIPTION
**Pull request by djegg**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92087)

### CODE REVIEW

**Comment by djegg** [comment]
@microsoft-github-policy-service agree [company="Winamp"]

**Comment by djegg** [comment]
@microsoft-github-policy-service agree company="Winamp"

**Comment by djegg** [comment]
NSIS installer

Silent install works fine

Winamp591_10029_final_full_en-us.exe /S

All dependencies (if missing) are configured to run silently.

File: manifests/w/Winamp/Winamp/5.90.0.9999/Winamp.Winamp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created with Notepad
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: "Winamp.Winamp"
+PackageVersion: "5.90.0.9999"
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 6.1.7601.0
+InstallerType: "nullsoft"
+Installers:
```
Comment (stephengillie) [inline manifests/w/Winamp/Winamp/5.90.0.9999/Winamp.Winamp.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
```

**Comment by stephengillie** [comment]
Hi @djegg,

This PR needs a few updates:
- Visual Studio Redistributable 2015+ x86 is a dependency. I've suggested the necessary dependency changes.
image
- PRs can only modify one manifest, but this one appears to modify both manifests for version 5.90.0.999 and version 5.91.0.10029. You mention the latter, so could you open another PR for the former and revert them here? 
image
- Less important: GitHub appears unhappy at the lack of line endings. 
image

**Comment by djegg** [comment]
Thanks Stephen. I'll make those changes shortly.

I didn't change anything for the previous 5.9.0 commit, so not sure why it ended up in this PR.
The PR should just be for 5.91.0.10029 only.

File: manifests/w/Winamp/Winamp/5.90.0.9999/Winamp.Winamp.installer.yaml (hunk 1 above)
Comment (djegg) [reply manifests/w/Winamp/Winamp/5.90.0.9999/Winamp.Winamp.installer.yaml:10]: 5.90.0.9999 folder is old and is not part of this PR.
No changes were made to any files in that old folder.

This PR is for new 5.91.0.10029 folder only.

**Comment by djegg** [comment]
Closing this ridiculous PR.
Starting again.

**Pull request closed without merging**